<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportMadXDialog</class>
 <widget class="QDialog" name="ImportMadXDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import from MADX</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>139</y>
     <width>157</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>65</y>
     <width>300</width>
     <height>64</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Particle Mass [GeV/c**2]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Total Energy [GeV]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxEtot">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="maximum">
       <double>1000000000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxM0c2">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="maximum">
       <double>10000000000000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEditFileName">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>26</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>5</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To: OptiM File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>363</x>
     <y>25</y>
     <width>89</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportMadXDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportMadXDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ImportMadXDialog</receiver>
   <slot>setOutputFilename()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setOutputFilename()</slot>
 </slots>
</ui>
